examples/openstack-lamp.yaml

```diff
@@ -56,52 +57,103 @@ inputs:
 
   site_yaml_relative_path:
     type: string
-    default: openvpn/openvpn.yml
+    default: ansible-examples/lamp_simple/site.yml
 
   key_name:
     type: string
 
+node_types:
+
+  webservers:
+    derived_from: cloudify.openstack.nodes.Server
+    properties: &vm_props
+      openstack_config:
+        default: *openstack_config
+      agent_config:
+        default:
+          install_method: none
+          key: { get_input: agent_key_private }
+          user: { get_input: username }
```

Comment:
@EarthmanT Since we have `install_method` do we need key, user values ?

Reply:
I see it as a good thing even if `install_method` is not used - for things like fabric and ansible plugin to get used to using this method.